Firestore/Example/Tests/Local/FSTWriteGroupTests.mm

```diff
@@ -80,26 +80,6 @@ - (void)testCommit {
   XCTAssertTrue(status.IsNotFound());
 }
 
-- (void)testDescription {
```

Comment:
You've deleted this test without replacing it. Batch descriptions are super useful in the log output when debugging so we can't delete this feature (and shouldn't delete its test).

Reply:
Will replace this and the following `TODO` after `descriptionForKey` in C++ lands.

Reply:
This seems like it could be restored?

Reply:
There is no longer a `WriteBatch` in `FSTWriteGroup`. I've deleted the code that constructed the string from it. The replacement for this test is in `FSTLevelDBTransactionTests`: `testToString`.